xdsl/printer.py

```diff
@@ -160,11 +163,12 @@ def _get_new_valid_block_id(self) -> int:
     def _print_result_value(self, op: Operation, idx: int) -> None:
         val = op.results[idx]
         self.print("%")
-        if val in self._ssa_values.keys():
```

Comment:
I kind of prefer with the `.keys`, since it's more explicit and more readable :/
What do you think?